release/models/bgp/openconfig-bgp-policy.yang

```diff
@@ -1290,6 +1319,184 @@ module openconfig-bgp-policy {
     }
   }
 
+  grouping set-as-path-inline-config {
+    description
+      "Configuration data for inline specification of
+      set-as-path action";
+
+    leaf-list as-paths {
+      type union {
+        type oc-inet:as-number;;
+        type bgp-as-mode-type;
+      }
+      description
+        "Set the AS path values for the update inline
+        with a list.";
+    }
+  }
+
+  grouping set-as-path-inline-state {
+    description
+      "Operational state data or inline specification of
+      set-as-path action";
+  }
+
+  grouping set-as-path-inline-top {
+    description
+      "Top-level grouping for inline specification of set-as-path
+      action";
+
+    container inline {
+      when "../config/method='INLINE'" {
+        description
+          "Active only when the set-as-path method is INLINE";
+      }
+      description
+        "Set the AS path values for the action inline with
+        a list.";
+
+      container config {
+        description
+          "Configuration data for inline specification of
+          set-as-path action";
+
+        uses set-as-path-inline-config;
+      }
+
+      container state {
+
+        config false;
+
+        description
+          "Operational state data or inline specification of
+          set-as-path action";
+
+        uses set-as-path-inline-config;
+        uses set-as-path-inline-state;
+      }
+    }
+  }
+
+  grouping set-as-path-reference-config {
+    description
+      "Configuration data for referening an as-path-set
+      in the set-as-path action";
+
+    leaf-list as-path-set-refs {
+      type leafref {
+        path "/oc-rpol:routing-policy/oc-rpol:defined-sets/" +
+          "oc-bgp-pol:bgp-defined-sets/" +
+          "oc-bgp-pol:as-path-sets/" +
+          "oc-bgp-pol:as-path-set/" +
+          "oc-bgp-pol:as-path-set-name";
+      }
+      description
+        "References a list of defined AS path sets by name";
+    }
+
+    // TODO(xqm): do we need to keep this deperecated leaf?
```

Comment:
@dplore Could you please help to confirm if we should keep this leaf which is deprecated in `set-community` and `set-ext-community`?

Reply:
Thanks for considering, but yes, no need to add a reference to deprecated leafs.    This can be removed.

Reply:
Removed `as-path-set-ref`. Thanks for the confirmation.